src/Servers/IIS/AspNetCoreModuleV2/InProcessRequestHandler/inprocessapplication.h

```diff
@@ -62,6 +64,18 @@ class IN_PROCESS_APPLICATION : public InProcessApplicationBase
     StopIncomingRequests()
```

Comment:
We were not blocking StopAsync before but are going to start now. Would it cause any issues? What if someone calls StopAsync on request thread?

Reply:
I think it is a problem.